src/context/layers/boundary.ts

```diff
@@ -1,4 +1,5 @@
 import { FeatureCollection } from 'geojson';
+// eslint-disable-next-line import/no-cycle
```

Comment:
As of Typescript 3.8, TS supports `import type { LayerDataParams } from './layer-data'`. Let's change type-only imports to that syntax instead of disabling eslint and see if eslint still complains in that case? 

[Reference](https://github.com/benmosher/eslint-plugin-import/issues/1453)